addons/account/tests/common.py

```diff
@@ -1,101 +1,81 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 from odoo import fields, Command
-from odoo.tests import Form, TransactionCase, HttpCase
+from odoo.tests import Form, HttpCase
 from odoo.tools.float_utils import float_round
 
+from odoo.addons.product.tests.common import ProductCommon
+
 import json
 import base64
 from lxml import etree
 from unittest import SkipTest
 
 
-def instantiate_accountman(cls):
-    cls.user = cls.env['res.users'].create({
-        'name': 'Because I am accountman!',
-        'login': 'accountman',
-        'password': 'accountman',
-        'groups_id': [
-            Command.set(cls.env.user.groups_id.ids),
-            Command.link(cls.env.ref('account.group_account_manager').id),
-            Command.link(cls.env.ref('account.group_account_user').id),
-        ],
-    })
-    cls.user.partner_id.email = 'accountman@test.com'
-
-    # Shadow the current environment/cursor with one having the report user.
-    # This is mandatory to test access rights.
-    cls.env = cls.env(user=cls.user)
-    cls.cr = cls.env.cr
-
-class AccountTestInvoicingCommon(TransactionCase):
+class AccountTestInvoicingCommon(ProductCommon):
 
     @classmethod
     def safe_copy(cls, record):
         return record and record.copy()
 
-    @classmethod
-    def copy_account(cls, account, default=None):
-        suffix_nb = 1
-        while True:
-            new_code = '%s.%s' % (account.code, suffix_nb)
-            if account.search_count([('company_id', '=', account.company_id.id), ('code', '=', new_code)]):
-                suffix_nb += 1
-            else:
-                return account.copy(default={**(default or {}), 'code': new_code, 'name': account.name})
+    @staticmethod
+    def setup_country(country_code):
```

Comment:
We do agree that we prefer this instead of just doing `cls.country_code = 'be'` before calling `super.setUpClass()`, right?